### ISSUE TITLE
Jetbrains editors SIGABRTing on master

### ISSUE THREAD

**Issue by timbess** [NixOS/nixpkgs#52302]
## Issue description
At least pycharm-community and idea-community are both SIGABRTing and dropping a core dump.

### Steps to reproduce
1. Install `jetbrains.pycharm-community`
2. run `pycharm-community`

## Technical details

Output:
```sh
[tbess@TimNixOS:~/src/nixpkgs]$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
[tbess@TimNixOS:~/src/nixpkgs]$ nix-build '.' -A jetbrains.pycharm-community
/nix/store/5q702amlcr4xl8pw6jl48qnc0bxlw3ak-pycharm-community-2018.3
[tbess@TimNixOS:~/src/nixpkgs]$ ./result/bin/pycharm-community

(java:9474): Gtk-WARNING **: 23:29:17.558: Unable to locate theme engine in module_path: "adwaita",

(java:9474): Gtk-WARNING **: 23:29:17.560: Unable to locate theme engine in module_path: "adwaita",
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f991a6389fd, pid=9474, tid=0x00007f989f9d4700
#
# JRE version: OpenJDK Runtime Environment (8.0_152-b06) (build 1.8.0_152-release-1248-b6)
# Java VM: OpenJDK 64-Bit Server VM (25.152-b6 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x1519fd]  __memmove_avx_unaligned_erms+0x1d
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/tbess/java_error_in_PYCHARM_9474.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
/nix/store/5q702amlcr4xl8pw6jl48qnc0bxlw3ak-pycharm-community-2018.3/pycharm-community-2018.3/bin/pycharm.sh: line 213:  9474 Aborted                 "$JAVA_BIN" -classpath "$CLASSPATH" ${VM_OPTIONS} "-XX:ErrorFile=$HOME/java_error_in_PYCHARM_%p.log" "-XX:HeapDumpPath=$HOME/java_error_in_PYCHARM.hprof" -Didea.paths.selector=PyCharmCE2018.3 "-Djb.vmOptionsFile=$VM_OPTIONS_FILE" ${IDE_PROPERTIES_PROPERTY} -Didea.platform.prefix=PyCharmCore com.intellij.idea.Main "$@"
```

I noticed when I run it with `strace` it tries to open a bunch of files that don't exist from the wrong bash derivation, then just SIGABRTs.

```sh
...... skipped to relevant portion .......
 
(java:9820): Gtk-WARNING **: 23:31:13.451: Unable to locate theme engine in module_path: "adwaita",

(java:9820): Gtk-WARNING **: 23:31:13.453: Unable to locate theme engine in module_path: "adwaita",
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fd00d4b49fd, pid=9820, tid=0x00007fcfea268700
#
# JRE version: OpenJDK Runtime Environment (8.0_152-b06) (build 1.8.0_152-release-1248-b6)
# Java VM: OpenJDK 64-Bit Server VM (25.152-b6 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x1519fd]  __memmove_avx_unaligned_erms+0x1d
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/tbess/java_error_in_PYCHARM_9820.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
[{WIFSIGNALED(s) && WTERMSIG(s) == SIGABRT}], 0, NULL) = 9820
rt_sigaction(SIGINT, {sa_handler=SIG_DFL, sa_mask=[], sa_flags=SA_RESTORER, sa_restorer=0x7f5e9df79c50}, {sa_handler=0x441490, sa_mask=[], sa_flags=SA_RESTORER, sa_restorer=0x7f5e9df79c50}, 8) = 0
openat(AT_FDCWD, "/nix/store/1mnsmslnx5anjfksac6417xfzzglrwhr-glibc-2.27/share/locale/locale.alias", O_RDONLY|O_CLOEXEC) = 3
fstat(3, {st_mode=S_IFREG|0444, st_size=2997, ...}) = 0
brk(0x82a000)                           = 0x82a000
read(3, "# Locale name alias data base.\n#"..., 4096) = 2997
brk(0x82b000)                           = 0x82b000
read(3, "", 4096)                       = 0
close(3)                                = 0
openat(AT_FDCWD, "/nix/store/6abyjgibafsbhlc7v7lab50mb3dj81jg-bash-4.4-p23/share/locale/en_US.UTF-8/LC_MESSAGES/bash.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/6abyjgibafsbhlc7v7lab50mb3dj81jg-bash-4.4-p23/share/locale/en_US.utf8/LC_MESSAGES/bash.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/6abyjgibafsbhlc7v7lab50mb3dj81jg-bash-4.4-p23/share/locale/en_US/LC_MESSAGES/bash.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/6abyjgibafsbhlc7v7lab50mb3dj81jg-bash-4.4-p23/share/locale/en.UTF-8/LC_MESSAGES/bash.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/6abyjgibafsbhlc7v7lab50mb3dj81jg-bash-4.4-p23/share/locale/en.utf8/LC_MESSAGES/bash.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/6abyjgibafsbhlc7v7lab50mb3dj81jg-bash-4.4-p23/share/locale/en/LC_MESSAGES/bash.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/1mnsmslnx5anjfksac6417xfzzglrwhr-glibc-2.27/share/locale/en_US.UTF-8/LC_MESSAGES/libc.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/1mnsmslnx5anjfksac6417xfzzglrwhr-glibc-2.27/share/locale/en_US.utf8/LC_MESSAGES/libc.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/1mnsmslnx5anjfksac6417xfzzglrwhr-glibc-2.27/share/locale/en_US/LC_MESSAGES/libc.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/1mnsmslnx5anjfksac6417xfzzglrwhr-glibc-2.27/share/locale/en.UTF-8/LC_MESSAGES/libc.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/1mnsmslnx5anjfksac6417xfzzglrwhr-glibc-2.27/share/locale/en.utf8/LC_MESSAGES/libc.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/nix/store/1mnsmslnx5anjfksac6417xfzzglrwhr-glibc-2.27/share/locale/en/LC_MESSAGES/libc.mo", O_RDONLY) = -1 ENOENT (No such file or directory)
write(2, "/nix/store/5q702amlcr4xl8pw6jl48"..., 470/nix/store/5q702amlcr4xl8pw6jl48qnc0bxlw3ak-pycharm-community-2018.3/pycharm-community-2018.3/bin/pycharm.sh: line 213:  9820 Aborted                 "$JAVA_BIN" -classpath "$CLASSPATH" ${VM_OPTIONS} "-XX:ErrorFile=$HOME/java_error_in_PYCHARM_%p.log" "-XX:HeapDumpPath=$HOME/java_error_in_PYCHARM.hprof" -Didea.paths.selector=PyCharmCE2018.3 "-Djb.vmOptionsFile=$VM_OPTIONS_FILE" ${IDE_PROPERTIES_PROPERTY} -Didea.platform.prefix=PyCharmCore com.intellij.idea.Main "$@"
) = 470
rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
--- SIGCHLD {si_signo=SIGCHLD, si_code=CLD_KILLED, si_pid=9820, si_uid=1000, si_status=SIGABRT, si_utime=116, si_stime=6} ---
wait4(-1, 0x7ffeee174e90, WNOHANG, NULL) = -1 ECHILD (No child processes)
rt_sigreturn({mask=[]})                 = 0
read(255, "", 6915)                     = 0
rt_sigprocmask(SIG_BLOCK, [CHLD], [], 8) = 0
rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
exit_group(134)                         = ?
+++ exited with 134 +++
```

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.
```sh
[tbess@TimNixOS:~/src/nixpkgs]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.9, NixOS, 18.09.1676.7e88992a8c7 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(tbess): `"home-manager"`
 - channels(root): `"nixos-18.09.1676.7e88992a8c7, unstable-19.03pre162671.44b02b52ea6"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by andywhite37** [comment]
I'm seeing this with idea-ultimate too

**Comment by tadfisher** [comment]
The 2018.3.1 updates have been merged into master; try that?

**Comment by enleth** [comment]
`unset GDK_PIXBUF_MODULE_FILE` before starting the IDE works around the problem. This is pretty much the same bug as #54278
